apps/meteor/app/lib/server/lib/notifyUsersOnMessage.js

```diff
@@ -106,7 +107,8 @@ export async function updateUsersSubscriptions(message, room) {
 		}
 
 		// this shouldn't run only if has group mentions because it will already exclude mentioned users from the query
-		if (!toAll && !toHere && unreadCount === 'all_messages') {
+		// and this should always run if it's a omnichannel room
+		if (!toAll && !toHere && (unreadCount === 'all_messages' || isOmnichannelRoom(room))) {
```

Comment:
so with this change, omnichannel rooms will always overwrite the `Unread_Count` setting value, therefore there will be no way of changing the behavior via configuration.. is that ok? what about creating a new setting `Unread_Count_Omni` to control this behavior? just an idea to be considered.

Reply:
That's a good point! I think we should make it configurable just like we're doing for rest of the rooms. Updated the PR and added this new setting. Please check :smiley: